.github/workflows/test-ubuntu.yml

```diff
@@ -76,40 +96,53 @@ jobs:
       - name: Install Pnpm
         run: corepack enable
 
-      - uses: dorny/paths-filter@v3
-        id: changes
-        with:
-          predicate-quantifier: 'every'
-          filters: |
-            changed:
-              - "!**/*.md"
-              - "!**/*.mdx"
-              - "!**/_meta.json"
-              - "!**/dictionary.txt"
-
       - name: Setup Node.js ${{ matrix.node-version }}
-        if: steps.changes.outputs.changed == 'true'
         uses: actions/setup-node@v4
         with:
           node-version: ${{ matrix.node-version }}
           cache: 'pnpm'
 
       - name: Install Dependencies
-        if: steps.changes.outputs.changed == 'true'
         run: pnpm install && cd ./tests && npx playwright install
 
       - name: Integration Test (Vitest)
-        if: steps.changes.outputs.changed == 'true'
         run: pnpm run test:integration
 
       - name: E2E Test (Playwright)
-        if: steps.changes.outputs.changed == 'true'
         run: pnpm run test:e2e
 
+  # ======== benchmark ========
+  benchemark-ubuntu:
+    runs-on: ubuntu-latest
+    needs: [integration-e2e-ubuntu]
+    strategy:
+      matrix:
+        # Only Ubuntu 20.04 and 22.04 are supported at the moment.
+        # See https://github.com/CodSpeedHQ/action/blob/016456b513677f9d4a1c509c7f8a38d8dd55b2b0/.github/workflows/ci.yml#L19.
+        node-version: [20.04]
```

Comment:
`node-version: [20.04]` will pin the `ubuntu-20.04`? 😲

Reply:
Yes, I misread it.